### PR TITLE
Adds trello link to README

### PR DESCRIPTION
**Pull request by sarony**
- Adds the trello link for our team project board so that everyone can find it.
- Trello card: https://trello.com/c/sjq9P0xc/10-i-want-to-see-which-letters-i-have-played-that-were-not-part-of-the-word-so-i-can-make-informed-guesses
- [ demo site, screen shots, sketches, etc. ]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+Trello link here: https://trello.com/b/ffCjlMki/js-tuesday
+
```
Comment (sarony) [inline README.md:2]: This link is awesome! I'm super impressed with your many skillz.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
 REBASE ME!
 
 All your re-Bass are belong to us
```
Comment (sarony) [inline README.md]: Interesting. Have you considered using a lower-case 'b'? I find that it makes your sentence clearer. [For an example of lowercases, see a book]

**Comment by sarony** [comment]
@sarony I made a few suggestions, but overall, this is looking great! Once you look at my comments, I'd love to hear what you think. Happy to take another look then.

File: README.md (hunk 1 above)
Comment (sarony) [inline README.md:2]: Thanks!

File: README.md (hunk 2 above)
Comment (sarony) [inline README.md]: That's a good idea. I will fix that next.

**Comment by sarony** [comment]
@sarony finished making the changes from your comment. Can you take a second look, and let me know if this is good to merge?

**Comment by sarony** [comment]
@sarony yup, this looks good to me :)

**Comment by mehulkar** [comment]
:rolleyes:

**Pull request merged**